.tool-versions

```diff
@@ -1,6 +1,6 @@
 # this file is used by asdf to automatically select the correct tool versions
 # but it also serves as a reference for non-asdf users
-uv
 terraform 1.6.6
 oc 4.12.46
 helm 3.11.1
+uv 0.6.10
```

Comment:
`0.6.11` is released now, you can bump it to match commit message

Reply:
lol this project and their release schedule :shaking_face: